azure-iptables-monitor/iptables_monitor.go

```diff
@@ -88,17 +111,16 @@ func patchLabel(clientset dynamic.Interface, labelValue bool, nodeName string) e
 	}
 	}`, label, labelValue))
 
-	_, err := clientset.Resource(gvr).
-		Patch(context.TODO(), nodeName, types.MergePatchType, patch, metav1.PatchOptions{})
+	err := clientset.PatchResource(context.TODO(), gvr, nodeName, types.MergePatchType, patch)
```

Comment:
We should create a context with some kind of timeout here. TODO is not meant for production code.